ext/src/ruby_api/store.rs

```diff
@@ -102,6 +114,20 @@ impl Store {
         self.context().data().user_data()
     }
 
+    /// @yard
+    /// Configure the WASI context on the store.
+    /// @def configure_wasi(wasi_config)
+    /// @param wasi_config [WasiConfig]
+    /// @return [Store] +self+
+    pub fn configure_wasi(
```

Comment:
If we want to be more explicit about what's going on under the hood maybe we can name this `set_wasi_from_config`.